src/main/java/CommandBye.java

```diff
@@ -0,0 +1,26 @@
+/**
+ * Represents the Command for "bye" for the Duke object. CommandBye helps to close and prepare Duke object for ending
+ * the run.
+ */
+public class CommandBye extends Command {
+    /**
+     * Closes the provided attributes of Duke object in preparation of ending the run.
```

Comment:
Perhaps an empty line between the description and parameters would fulfil the code's adherence to standards regarding Javadoc? ... I've also noticed the same trait across many Javadoc comments in most classes